src/python/grpcio/tests/protoc_plugin/beta_python_plugin_test.py

```diff
@@ -244,35 +244,63 @@ class PythonPluginTest(unittest.TestCase):
   methods and does not exist for response-streaming methods.
   """
 
-  def setUp(self):
```

Comment:
Huh? What's the reason for replacing `setUp` and `tearDown` with `__init__` and `__del__`?